src/test/java/org/opensearch/performanceanalyzer/config/PerformanceAnalyzerControllerTests.java

```diff
@@ -53,8 +53,9 @@ public void init() {
 
     @Test
     public void testGetNodeStatsShardsPerCollection() {
-        assertEquals(
-                DEFAULT_NUM_OF_SHARDS_PER_COLLECTION, controller.getNodeStatsShardsPerCollection());
+        Assert.assertEquals(
```

Comment:
Should we keep an uniform format here in the package-wise? (either use Assert.assertEquals or assertEquals)

Reply:
Right. Assert.assertEquals is used almost everywhere. In case of ConfigOverridesIT.java, it extends that from the PerformanceAnalyzerIntegTestBase. Spotless takes care of these changes automatically.